contrib/internetConnected.py

```diff
@@ -1,19 +1,10 @@
 
-import socket
-def internetConnected(host="8.8.8.8", port=53, timeout=3):
-   """
-   Simple function to quickly check, are we connected to the internet.
-   Needed to detect when waking up from sleep or returning from suspend.
-   Network can take a few seconds to reconnect. Sometimes longer.
-   Default connect to host: 8.8.8.8 (google-public-dns-a.google.com)
-   OpenPort: 53/tcp
-   Service: domain (DNS/TCP)
-   Average time to check is less than 0.2 second (200 ms).
-   """
+import requests
```

Comment:
`requests` is not from the standard library. This means we add a new dependency. I'm not sure we want to forcefully require a new dependency, here. We might better like to make use of it if present or fallback to something else.

Anyway, this must be documented in the README.